package.json

```diff
@@ -4,18 +4,19 @@
   "description": "Demo of React Hot Reloading + TypeScript",
   "main": "./dist/bundle.js",
   "dependencies": {
-    "react": "^15.3.2",
-    "react-dom": "^15.3.2"
+    "tslib": "1.2.0",
```

Comment:
`tslib` is needed to reuse all the helper code, e.g "__extends" for `class ... extends`, "__awaiter" for `async`/`await`